app/manager-frontend/src/pages/ActivitySource/index.jsx

```diff
@@ -87,42 +110,41 @@ export const ActivitySource = ({ activitySource, action, id }) => {
   };
 
   return (
-    <Container my={8} ref={scrollTarget}>
-      <VStack w="100%" align="stretch" spacing={4} p={4} pb={10}>
+    <Container my={8} ref={scrollTarget} style={{ maxWidth: "850px" }}>
+      <VStack align="stretch" spacing={4} p={4} pb={10}>
         <Flex justify="space-between">
-          <Heading>{headingText}</Heading>
-          {id && (
-            <Button
-              leftIcon={<FaTrash />}
-              variant="outline"
-              colorScheme="red"
-              onClick={onOpen}
-            >
-              Delete
-            </Button>
-          )}
+          {headingText}
+          <Button
+            leftIcon={<FaTimes />}
+            variant="outline"
+            colorScheme="red"
```

Comment:
```suggestion
            colorScheme="gray"
```

Cancel buttons shouldn't be red; they're not dangerous or destructive, and you often have a red delete button in the same context (form in this case). They should be grey and unassuming.